skypy/galaxy/luminosity.py

```diff
@@ -54,47 +49,32 @@ def herbel_luminosities(redshift, alpha, a_m, b_m, size=None,
     Notes
     -----
     The Schechter luminosity function is given as
-
     .. math::
-
         \Phi(L, z) = \frac{\Phi_\star(z)}{L_\star(z)}
             \left(\frac{L}{L_\star(z)}\right)^\alpha
             /exp\left(-\frac{L}{L_\star(z)}\right) \;.
-
     Here the luminosity is defined as
-
     .. math::
-
         L = 10^{-0.4M} \;,
-
     with absolute magnitude :math:`M`. Furthermore, Herbel et al. [1]_
     introduced
-
     .. math::
-
         \Phi_\star(z) = b_\phi \exp(a_\phi z) \;,
         M_\star(z) = a_M z + b_M \;.
-
     Now we have to rescale the Schechter function by the comoving element and
     get
-
     .. math::
-
         \phi(L,z) = \frac{d_H d_M^2}{E(z)}  \Phi(L,z)\;.
 
     Examples
     --------
     >>> import skypy.galaxy.luminosity as lum
-
-    Sample 100 luminosity values at redshift z = 1.0 with
-    a_m = -0.9408582, b_m = -20.40492365, alpha = -1.3.
-
+    >>> # Sample 100 luminosity values at redshift z = 1.0 with
+    >>> # a_m = -0.9408582, b_m = -20.40492365, alpha = -1.3.
```

Comment:
The original formatting including the blank lines was correct. We want the rendered documentation to be a mix of text and command line. Please undo this change.

```suggestion

    Sample 100 luminosity values at redshift z = 1.0 with
    a_m = -0.9408582, b_m = -20.40492365, alpha = -1.3.

```